src/services/player/player.js

```diff
@@ -3,11 +3,24 @@ export default class Player {
         this.playerName = playerName;
         this.points = points;
         this.id = Date.now();
+        this.question = [];
+        this.answers = [];
+        this.correctAnswers = [];
     }
 
     save() {
         localStorage.setItem('playerName', this.playerName);
         localStorage.setItem('points', this.points);
     }
 
+    getQuestion(callback, gameMode) {
```

Comment:
do you mind if instead of passing callback function to `getQuestion` method, we will just use this possible callback

I am not sure if this method will be reusable so we can just use (AFAIK) `generateQuestion` function